detox/e2e/test/messaging/tap_send_button_repeatedly.e2e.js

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import nodeExpect from 'expect';
+import jestExpect from 'expect';
```

Comment:
Corrected the name since it's actually Jest, and not node. Will help especially when one would like to look/search for `expect` documentation online.